pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus-ha
 hubproj=citus-ha
 pkgdesc='Citus HA'
-pkglatest=1.0.0
-releasepg=11
+pkglatest=2.0.0
+releasepg=10,11
```

Comment:
Are you sure this works? (package build fails for me unless I change this to `releasepg=11`)

Reply:
I was hoping this worked, as we do it in the build scripts of citus as well.
`10` and `11` build separately, but apparently not together.

I think we want to release both versions of citusha, so I guess we need to land a 10 build separately in that case.